src/components/PrivateRoute.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Redirect, Route } from 'react-router-dom'
+
+function PrivateRoute ({component: Component, authenticated, ...rest}) {
+  return (
+    <Route
+      {...rest}
+      render={(props) => authenticated === true
+        ? <Component {...props} />
+        : <Redirect to={{pathname: '/login', state: {from: props.location}}} />}
```

Comment:
I don't think we need to send in `state: {from: props.location}`, we're not using it yet.